### DISCUSSION TITLE
Email deliverability issues (Emails go to spam)

### DISCUSSION THREAD

**Discussion by virajsoni06** [swellstores/community#535] [Help]
Hi, any email sent from our store goes to spam. I see you have SendGrid integration but we don't use sendgrid and don't intend to as we already are using Postmark. What's the issue with sending emails from Swell and how can that be resolved?

**Comment by logeshswell** [comment]
1. Does the email sent from Swell or Postmark?
2. Can you specify the email notification type from the Swell admin dashboard for which it moves to Spam?
3. Screenshot of the sender's email would also be very helpful in debugging this issue.

**Reply by logeshswell** [reply]
Usually, SPF and DKIM policies in the DNS server cause the emails to move to **Spam** folder.

**Reply by logeshswell** [reply]
I see your SPF record allows only Postmark emails to be sent using your custom domain.
Rest of the emails are marked as Spam.

**Reply by logeshswell** [reply]
To validate it, you can delete the SPF record and try sending test emails in Swell store.
If it works fine, then you can include `swell.store` in your SPF record for your custom domain in the DNS configuration.

**Reply by virajsoni06** [reply]
@logeshswell emails are sent from Swell, I was referring to the service we use for transactional mail, which infact works fine.
Our DKIM/SPF is already setup correctly to send emails but since Swell uses its own emailing service, SendGrid, we don't have access to its DKIM config to set it up.
Can you please let me know how can the swell mail servers can be verified so that emails from Swell works?

**Reply by virajsoni06** [reply]
> To validate it, you can delete the SPF record and try sending test emails in Swell store. If it works fine, then you can include `swell.store` in your SPF record for your custom domain in the DNS configuration.

@logeshswell added swell.store to SPF record, still doesn't land in mailbox

**Reply by logeshswell** [reply]
Can you also add your subdomain (`info@exambeacon.co.uk`) to the DMARC policy?

**Reply by virajsoni06** [reply]
@logeshswell which subdomain? You've mentioned the email address in previous message

**Reply by logeshswell** [reply]
Sorry, I mean, can you add a "FROM" email address (info@exambeacon.co.uk) to your DMARC policy?
Because your DMARC record is enabled with "quarantine" policy. So, the email addresses that don't match the DMARC policy will be sent to the Spam folder.
For more info, please take a look:
https://easydmarc.com/tools/dmarc-lookup?domain=exambeacon.co.uk

**Reply by virajsoni06** [reply]
@logeshswell doesn't work. Have added info@exambeacon.co.uk as an entry in rua.
![image](https://github.com/swellstores/community/assets/13236286/2e54f263-7b36-4adb-9720-280a30709135)

PS: Don't think we can add a "from" email in dmarc

**Comment by virajsoni06** [comment]
@logeshswell can you look into this please

**Reply by logeshswell** [reply]
Since you updated the DMARC policy type as `p=none`, I am getting the email delivered in **Inbox**.
Can you also check and confirm?

**Reply by virajsoni06** [reply]
@logeshswell yes the emails are being delivered into inbox but having DMARC set as none is highly insecure as emails can be spoofed. Its standard practice to not have `p=none` for DMARC and that's not something we should even have. There needs to be a way where either we should be able to configure our mail servers directly on Swell or if Swell is handling it, it should do it properly and not have such attack vectors exposed.

**Reply by logeshswell** [reply]
Yes, you are totally right. To secure your email address from spoofing, you may prefer to use the Sendgrid integration present in Swell. Sendgrid takes care of these security measures.

**Reply by virajsoni06** [reply]
@logeshswell see the first message. As I had mentioned before, we don't have Sendgrid and don't want to use it as we already are using Postmark. Sendgrid isn't going to work

**Reply by logeshswell** [reply]
Then, please try using Postmark with similar features:
https://postmarkapp.com/guides/agencies-sending-on-behalf-of-users
